src/main/app/src/Components/EventComponent/Events.tsx

```diff
@@ -36,7 +40,7 @@ const Events: React.FC<{}> = () => {
     content = data.messages.map((m, i) => {
       return (
         <div className={'item'} key={i}>
-          <Chip label={m.landscape} /> {m.event}
+          {m.date} <LevelChip level={m.level} title={m.landscape}/> {m.type} {m.message}
```

Comment:
hier evtl noch nen span drum, damit das date nicht im dom schwebt

Reply:
For my understanding, what is wrong about it?

Reply:
its more about conventions here: https://html.spec.whatwg.org/multipage/grouping-content.html#the-div-element 

"Authors are strongly encouraged to view the div element as an element of last resort, for when no other element is suitable. Use of more appropriate elements instead of the div element leads to better accessibility for readers and easier maintainability for authors."